File: acm_statistic_backend/api/api/DTO/evolutionTemporel/RedevanceMensuelDTO.cs
namespace api.DTO.evolutionTemporel
{
    public class RedevanceMensuelDTO
    {
        public int Mois {  get; set; }
        public string Mois_lettre { get; set; }

        public int Annee { get; set; }

        public String? Type_trafic {  get; set; }
        public decimal? Montant { get; set; }
        public string? Compagnie { get; set; }
        public string? Aeroport { get; set; }
    }
}

File: acm_statistic_frontend/Interface Tableau de Bord Analytique Aérien/src/models/evolutionTemporel/RedevanceMensuelDTO.ts
export class RedevanceMensuelDTO{
    mois: number;
    mois_lettre: string | null;
    annee :number; 
    type_trafic: string | null;
    montant: number;
    compagnie: string | null;
    aeroport: string | null;

    constructor(init?: Partial<RedevanceMensuelDTO>) {
    Object.assign(this, init);
    }

}